settings.py

```diff
@@ -1,6 +1,7 @@
 from pathlib import Path
 
 DEBUG = True
+TESTING = True
```

Comment:
TODO: should be useless when merging from master with dedicated testing setting file.

Reply:
Actually, no. I think it's related to the way we load app's extension in the global `__init__` before we set the testing settings.

Reply:
Done for mail, cannot be done for debug toolbar though without a [better organization of the project](https://github.com/stephane/flask-skeleton/blob/master/skeleton/__init__.py).